.prettierrc.json

```diff
@@ -17,12 +17,6 @@
         "parser": "css",
         "singleQuote": false
       }
-    },
-    {
-      "files": "*.md",
```

Comment:
We're no longer running Prettier on our .md files? Can we tweak the config instead?